docs/source/yelpsoa_configs.rst

```diff
@@ -1072,7 +1072,7 @@ internally at Yelp.
      integer, this cannot be specified as a floating-point number.
 
 Error Alerting
-``````````````
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
what's the difference? between the backtick and the `^` ?

Reply:
https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst#section-structure

Reply:
There are some formatting errors with the rst that makes the readthedocs a bit wrong, i changed it to see if it will help